Makefile

```diff
@@ -12,7 +12,7 @@ sync:
 	git submodule update --init --recursive
 
 ocamlformat:
-	# Use 2de7c5f version of ocamlformat
+	# Use ocamlformat.0.9
```

Comment:
you can use version matching in the `.ocamlformat` file to enforce that (instead of the comment)

Reply:
Done!